README.md

```diff
@@ -319,6 +319,7 @@ For authentication you can also use Travis CI secure environment variable:
 * **region**: S3 Region. Defaults to us-east-1.
 * **upload-dir**: S3 directory to upload to. Defaults to root directory.
 * **storage-class**: S3 storage class to upload as. Defaults to "STANDARD". Other values are "STANDARD_IA" or "REDUCED_REDUNDANCY". Details can be found [here](https://aws.amazon.com/s3/storage-classes/).
+* **server-side-encryption**: When set to `true`, use S3 Server Side Encryption (SSE-AES256). Defaults to non-encrypted.
```

Comment:
Very sorry about the README. I did make the change, but it looks like I didn't push it to the branch. Doh! Thanks for taking care of that and merging, @BanzaiMan!